contracts/Vault.sol

```diff
@@ -34,7 +51,8 @@ contract Vault is Initializable, UUPSUpgradeable, AccessControlUpgradeable, Reen
     mapping(uint256 => bool) public fillNonce;
     mapping(uint256 => bool) public settleNonce;
     bytes32 private constant UPGRADER_ROLE = keccak256("UPGRADER_ROLE");
-
+    bytes32 private constant SETTLEMENT_VERIFIER_ROLE =
+        keccak256("SETTLEMENT_VERIFIER_ROLE");
     // Storage gap to reserve slots for future use
     uint256[50] private __gap;
```

Comment:
note: storage slot need not be reduced as constants are part of bytecode not storage